toolchain/semantics/semantics_ir.cpp

```diff
@@ -128,6 +128,22 @@ auto SemanticsIR::StringifyNodeImpl(llvm::raw_ostream& out,
 
   auto node = GetNode(node_id);
   switch (node.kind()) {
+    case SemanticsNodeKind::StructType: {
+      out << "{";
+      auto refs = GetNodeBlock(node.GetAsStructType().second);
+      llvm::ListSeparator sep;
+      for (const auto& ref_id : refs) {
+        out << sep;
+        StringifyNodeImpl(out, ref_id);
```

Comment:
A long way from mattering, but maybe worth leaving a todo for when we end up wanting a fuzzer to not waste time here...

```suggestion
        // TODO: Bound recursion depth or remove recursive step.
        StringifyNodeImpl(out, ref_id);
```